applets/wncklet/window-list.c

```diff
@@ -608,6 +611,15 @@ static void move_unminimized_windows_changed(GSettings* settings, gchar* key, Ta
 	tasklist_update_unminimization_radio(tasklist);
 }
 
+static void scroll_enabled_changed (GSettings* settings, gchar* key, TasklistData* tasklist)
+{
+	gboolean value;
+
+	value = g_settings_get_boolean (settings, key);
+	tasklist->scroll_enable = (value != 0);
```

Comment:
why not `tasklist->scroll_enable = g_settings_get_boolean (settings, key);`?

Reply:
Has been updated